frontend/src/views/OfferView.vue

```diff
@@ -29,13 +33,28 @@ let offerInfo = ref<OfferCreate | Offer>({
   items: [],
 });
 
+let offerItemsArray = ref<any>([]);
+
+function addOfferItem(type: ArticleKind) {
+  offerItemsArray.value.push({ id: id++, type: type });
+}
+
 onMounted(async () => {
   orderInfo.value = await getOrder(route.params.order_id as string);
 });
 </script>
 
 <template>
-  <div v-if="orderInfo" class="my-2">
+  <div class="flex flex-row justify-between">
+    <router-link :to="`${ROUTES.ORDER.path}/${orderInfo?.id}/edit`">
```

Comment:
Why can there be no orderInfo?

Reply:
We receive it async so there could be no orderInfo.